src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/ReadDescriptorsValidation.feature

```diff
@@ -16,24 +16,6 @@ Feature: Read a Descriptor
                   """
              Then it should respond with 201 or 200
 
-        Scenario: 01 Verify existing descriptors can be retrieved successfully
-             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors"
-             Then it should respond with 200
-              And the response body is
-                  """
-                    [
-                        {
-                            "id": "{id}",
-                            "codeValue": "Sick Leave",
-                            "description": "Sick Leave",
-                            "effectiveBeginDate": "2024-05-14",
-                            "effectiveEndDate": "2024-05-14",
-                            "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
-                            "shortDescription": "Sick Leave"
-                        }
-                    ]
-                  """
-
```

Comment:
@msilesgap when we merge this PR, some of the scenarios will move. We will need to update the spreadsheet accordingly.

Why? Suganya and I discussed and agreed that this would simplify our ability to only reset OpenSearch between tests if really needed. Thus we want to keep all of the OpenSearch-related tests together in one place. We thought about using tags, but there is also a setup step that needs to happen ("background"), and we did not want that setup step scattered in many different places.